android/src/main/java/com/theoplayer/presentation/PresentationManager.kt

```diff
@@ -229,7 +232,10 @@ class PresentationManager(
         hide(WindowInsetsCompat.Type.systemBars())
       }
 
-      updatePresentationMode(PresentationMode.FULLSCREEN)
+      // Delay the event making sure it does not arrive before animations ended.
+      Handler(Looper.getMainLooper()).postDelayed({
+        updatePresentationMode(PresentationMode.FULLSCREEN)
+      }, FULLSCREEN_EVENT_DELAY)
```

Comment:
```suggestion
      // Delay the event making sure it does not arrive before animations ended.
        viewCtx.playerView.postOnAnimationDelayed(FULLSCREEN_EVENT_DELAY) {
            updatePresentationMode(PresentationMode.FULLSCREEN)
        }
```

given `import androidx.core.view.postOnAnimationDelayed`

Reply:
or mb even this would be sufficient?
```suggestion
      // Delay the event making sure it does not arrive before animations ended.
        viewCtx.playerView.postOnAnimation {
            updatePresentationMode(PresentationMode.FULLSCREEN)
        }
```

Reply:
Thanks Philipp, this looks way cleaner indeed 👍